src/reducers/reducer.js

```diff
@@ -912,15 +916,22 @@ const onSetUseWalletService = (state, action) => ({
   useWalletService: action.payload,
 });
 
+const onSetUseSafeBiometryMode = (state, action) => ({
+  ...state,
+  safeBiometryEnabled: action.payload,
+});
+
 const onResetWalletSuccess = (state) => {
   const oldUnleashClient = state.unleashClient;
   const oldFeatureTogglesInitialized = state.featureTogglesInitialized;
   const oldFeatureToggles = state.featureToggles;
+  const oldSafeBiometryEnabled = state.safeBiometryEnabled;
```

Comment:
Yes, but this is the state of the feature flag, so that we can catch toggles.

From what I understand the `state.featureToggles` is also here because the feature sagas do not fetch and set the state on the restart so we need to keep it unchanged when resetting the state.

Reply:
Why don't you keep this new one together with the other feature toggles?

Reply:
It is kept together, but to identify that it has changed we need to keep a separate copy so that after the `featureToggles` are updated we can compare the 2.